packages/tangle/timemanager.go

```diff
@@ -136,7 +144,7 @@ func (t *TimeManager) synced() bool {
 func (t *TimeManager) updateSyncedState() {
```

Comment:
We don't need the `Synced` and `updateSyncedState` in their current form anymore right? We just want to have a concept of bootstrapped and a concept of synced that just checks a threshold between FTT and RCTT.

Reply:
I thought we leave the synced condition as is and just update it once. When the node becomes the first time synced, it means it bootstrapped, and we don't update synced status anymore.

Reply:
The bootstrapped flag is the only one that affects algorithms: until the flag turns true we won't emit messages (blocks in the new terminology). This flag won't transition to false once set to true.
The synced flag instead has no effect in the code whatsoever, and it is purely a user-facing value. This flag can transition back and forth between true and false (as it is now).

Reply:
@daria305 can you participate?